bin/eds-import-from-figma-api.js

```diff
@@ -124,33 +123,58 @@ const { identity } = require('lodash');
     process.exit(-1);
   }
 
-  const edsCollection = figmaApiReader
+  // get the data for the full collection from the API reader
+  const tier1Collection = figmaApiReader
     .getVariableCollections()
+    .filter((collection) => !collection.hiddenFromPublishing)
+    .find(
+      (collection) => collection.id === collectionNameResponses.tier1Collection,
+    );
+
+  const tier2Collection = figmaApiReader
+    .getVariableCollections()
+    .filter((collection) => !collection.hiddenFromPublishing)
     .find(
       (collection) => collection.id === collectionNameResponses.tier2Collection,
     );
 
   // Determine which of the modes in the file should be used
-  let response;
+  let modeResponses;
   try {
-    response = await prompt({
```

Comment:
From each collection we should select the mode. in the current tier 1 collection, we only have the one mode to pick, for tier 2, the mode represents the theme to use